subseas_flex_fcst/maproom.py

```diff
@@ -453,8 +453,7 @@ def fcst_tiles(tz, tx, ty, proba, variable, percentile, threshold, startDate,lea
         coords = fcst_mu.rename({"X": "lon", "Y": "lat"}).coords,
         dims = fcst_mu.rename({"X": "lon", "Y": "lat"}).dims
     # pingrid.tile wants 2D data
-    ).squeeze("T")
-    print("test2")
+    )#.squeeze("T")
```

Comment:
Perhaps because of how the data is combined, the need to remove the T dimension is not required? Not sure exactly but it seems to work okay without it....